package.json

```
@@ -112,24 +113,24 @@
     "dedent": "^1.5.1",
     "eslint": "^8.56.0",
     "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-svelte": "^2.35.1",
+    "eslint-plugin-svelte": "^2.46.0",
     "highlight.js": "^11.7.0",
     "highlightjs-svelte": "^1.0.6",
     "husky": "^8.0.3",
     "just-clamp": "^4.2.0",
     "lint-staged": "^14.0.1",
     "postcss": "^8.4.14",
-    "prettier": "^3.2.5",
-    "prettier-plugin-svelte": "^3.1.2",
+    "prettier": "^3.3.3",
+    "prettier-plugin-svelte": "^3.2.7",
     "prettier-plugin-tailwindcss": "^0.5.11",
-    "publint": "^0.1.16",
-    "svelte": "^4.0.0",
-    "svelte-check": "^3.6.4",
+    "publint": "^0.2.10",
+    "svelte": "^5.1.9",
+    "svelte-check": "^4.0.5",
     "tailwindcss": "^3.2.7",
```

I had to upgrade this to get Skeleton v2 working on my branch
```suggestion
    "tailwindcss": "^3.4.14",
```